src/java.xml/share/classes/module-info.java

```diff
@@ -27,6 +27,236 @@
+ * setting security features to true and limits to the defined values shown in
+ * the table. The property values will not be affected, however, when setting
+ * secure processing to false.
+ * <p>
+ * When the Java Security Manager is present, secure processing is set to true
+ * and can not be turned off. The security properties are therefore enforced.
+ * <p>
+ * Properties specified in the jaxp.properties file affect all invocations of
+ * the JDK, and will override their default values, or those that may have been
+ * set by secure processing.
+ * <p>
+ * System properties, when set, affect the invocation of the JDK and override
+ * the default settings or those that may have been set in jaxp.properties or
+ * by secure processing.
+ * <p>
+ * JAXP properties specified through JAXP factories or processors (e.g. SAXParser)
+ * take preference over system properties, the jaxp.properties file, as well as
+ * secure processing.
+ * <p>
+ *
+ * <h3 id="Processor">Processor Support</h3>
+ * Features and properties may be supported by one or more processors. The
+ * following table lists the processors by IDs that can be used for reference.
+ * <p>
+ *
+ * <table class="plain" id="Processors">
+ * <caption>Processors</caption>
+ * <thead>
+ * <tr>
+ * <th scope="col">ID</th>
+ * <th scope="col">Name</th>
+ * <th scope="col">How to set the property</th>
+ * </tr>
+ * </thead>
+ *
+ * <tbody>
+ * <tr>
+ * <th scope="row" style="font-weight:normal" id="DOM">DOM</th>
+ * <td>DOM Parser</td>
+ * <td>
+ * DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();<br>
+ * dbf.setAttribute(name, value);
+ * </td>
+ * </tr>
+ * <tr>
+ * <th scope="row" style="font-weight:normal" id="SAX">SAX</th>
+ * <td>SAX Parser</td>
+ * <td>
+ * SAXParserFactory spf = SAXParserFactory.newInstance();<br>
+ * SAXParser parser = spf.newSAXParser();<br>
+ * parser.setProperty(name, value);
+ * </td>
+ * </tr>
+ * <tr>
+ * <th scope="row" style="font-weight:normal" id="StAX">StAX</th>
+ * <td>StAX Parser</td>
+ * <td>
+ * XMLInputFactory xif = XMLInputFactory.newInstance();<br>
+ * xif.setProperty(name, value);
+ * </td>
+ * </tr>
+ * <tr>
+ * <th scope="row" style="font-weight:normal" id="Validation">Validation</th>
+ * <td>XML Validation API</td>
+ * <td>
+ * SchemaFactory schemaFactory = SchemaFactory.newInstance(schemaLanguage);<br>
+ * schemaFactory.setProperty(name, value);
+ * </td>
+ * </tr>
+ * </tr>
+ * <tr>
+ * <th scope="row" style="font-weight:normal" id="Transform">Transform</th>
+ * <td>XML Transform API</td>
+ * <td>
+ * TransformerFactory factory = TransformerFactory.newInstance();<br>
+ * factory.setAttribute(name, value);
+ * </td>
+ * </tr>
+ * </tr>
+ * <tr>
+ * <th scope="row" style="font-weight:normal" id="DOMLS">DOMLS</th>
+ * <td>DOM Load and Save</td>
+ * <td>
+ * LSSerializer serializer = domImplementation.createLSSerializer(); <br>
+ * serializer.getDomConfig().setParameter(name, value);
+ * </td>
+ * </tr>
+ * </tbody>
+ * </table>
+ *
```

Comment:
One question is whether the code samples in the table above should be escaped with {@code }. e.g.:

```
<td>
{@code first line of code;}<br>
{@code second line of code;}<br>
</td>
```

Reply:
Looks better.